Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=b16363991414a6025beb7269f9c1dd294f9b241f
Commit:     b16363991414a6025beb7269f9c1dd294f9b241f
Parent:     e2673b28911a43257265523e3672861be6e44093
Author:     Oliver Neukum <oliver@neukum.org>
AuthorDate: Fri Dec 12 11:01:45 2008 +0100
Committer:  Greg Kroah-Hartman <gregkh@suse.de>
CommitDate: Wed Jan 7 09:59:48 2009 -0800

    USB: storage: extend unusual range for 067b:3507
    
    This device has been released in a new revision which is still buggy.
    
    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Phil Dibowitz <phil@ipom.com>
    Cc: stable <stable@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/usb/storage/unusual_devs.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/storage/unusual_devs.h b/drivers/usb/storage/unusual_devs.h
index af34504..a7b2db7 100644
--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -1040,7 +1040,7 @@ UNUSUAL_DEV( 0x067b, 0x2507, 0x0100, 0x0100,
 		US_FL_FIX_CAPACITY | US_FL_GO_SLOW ),
 
 /* Reported by Alex Butcher <alex.butcher@assursys.co.uk> */
-UNUSUAL_DEV( 0x067b, 0x3507, 0x0001, 0x0001,
+UNUSUAL_DEV( 0x067b, 0x3507, 0x0001, 0x0101,
 		"Prolific Technology Inc.",
 		"ATAPI-6 Bridge Controller",
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
